src/tusb_option.h

```diff
@@ -267,6 +267,15 @@
   #define CFG_TUD_DWC2_DMA_ENABLE CFG_TUD_DWC2_DMA_ENABLE_DEFAULT
 #endif
 
+// Enable CI_HS VBUS Charge
```

Comment:
maybe add the description on why this option is needed e.g when USB_VBUS pin is not connected to 5V

Reply:
Comment just added